### PR TITLE
Update translations as per documentation

### PR DESCRIPTION
**Pull request by garethbowen**
I mistakenly updated the translations in POE as per the old
instructions. The new way to modify the English values is directly
in the properties file to be imported into POE before release.

This updates them correctly for the following issues

medic/cht-core#5383
medic/cht-core#5392
medic/cht-core#5394

# Code review items

- Readable: Concise, well named, follows the [style guide](https://github.com/medic/medic-docs/blob/master/development/style-guide.md), documented if necessary.
- Documented: Configuration and user documentation on [medic-docs](https://github.com/medic/medic-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in Changes.md.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
Code review please @kennsippell 

No AT is necessary as these have been ATed manually already.

File: ddocs/medic/_attachments/translations/messages-en.properties [hunk 1]
```diff
@@ -1020,12 +1020,12 @@ sent = Sent timestamp
 sent_timestamp = Sent timestamp
 settings.backup.action = Download current settings
 settings.backup.description = Download a copy of the current settings.
-settings.backup.title = Backup current settings
-settings.backuprestore = Backup/Restore
-settings.backuprestore.description = Here you can download or upload the application settings in JSON format.
-settings.restore.action = Upload new settings
+settings.backup.title = Backup application code
+settings.backuprestore = Backup app code
```
Comment (kennsippell) [inline ddocs/medic/_attachments/translations/messages-en.properties]: used "application code" everwhere else but "app code" here. Unsure if there is a reason for the inconsistency?

**Comment by kennsippell** [comment]
@gareth One comment that is probably nothing - but take a look and I'll let you handle the merge.

File: ddocs/medic/_attachments/translations/messages-en.properties (hunk 1 above)
Comment (garethbowen) [reply ddocs/medic/_attachments/translations/messages-en.properties]: That's used in the nav sidebar so it's abbreviated so it doesn't line wrap.

**Pull request merged**